ersilia/hub/content/catalog.py

```diff
@@ -37,10 +38,40 @@ def as_list_of_dicts(self):
                 d[c] = r[i]
             R += [d]
         return R
-
+    
     def as_json(self):
         R = self.as_list_of_dicts()
         return json.dumps(R, indent=4)
+    
+    def generate_separator_line(self, left, middle, right, horizontal, widths):
+        return left + middle.join(horizontal * (width + 2) for width in widths) + right
+
+    def as_table(self):
+        column_widths = [
```

Comment:
@jubril-adeyi apologies for dragging this out - just few minor things before we close this PR:
Could you add a docstring for the `as_table` method to explain your logic?
Also please format the code with black - this is extending far beyond the ideal 80 column width for python programs.

Reply:
@DhanshreeA docstring has been added for the as_table method and the seperator_line helper method, explaining the logic. 
The code has also been formated using black, in the latest commit